infra/base-images/base-builder/compile_gfuzztests.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+bazel run @com_google_fuzztest//bazel:setup_configs > fuzztest.bazelrc
+
+# Bazel target names of the fuzz binaries
+FUZZ_TEST_BINARIES=$(bazel query 'kind("cc_test", rdeps(..., @com_google_fuzztest//fuzztest:fuzztest_gtest_main))')
+
+# Bazel output paths of the fuzz binaries
+FUZZ_TEST_BINARIES_OUT_PATHS=$(bazel cquery 'kind("cc_test", rdeps(..., @com_google_fuzztest//fuzztest:fuzztest_gtest_main))' --output=files)
+
+#echo "Fuzz test binaries:"
+#echo ${FUZZ_TEST_BINARIES}
+#echo "-----------------------------"
+#echo ${FUZZ_TEST_BINARIES3}
+#echo "-----------------------------"
+
+# Build the project and fuzz binaries
+bazel build --subcommands --config=oss-fuzz ${FUZZ_TEST_BINARIES[*]}
+
+# Iterate the fuzz binaries and list each fuzz entrypoint in the binary. For
+# each entrypoint create a wrapper script that calls into the binaries the
+# gien entrypoint as argument.
+# The scripts will be named:
+# {binary_name}_GFUZZTEST_{fuzztest_entrypoint}
```

Comment:
Is the GFUZZTEST part necessary?

If we keep it, consider dropping the G here too.

Reply:
Yes, it's necessary in that we need to identify when a fuzz target is in the form of a shell script that calls into a fuzztest binary. We could add this token in other ways (e.g. a tag in script), but am happy to leave it as is after dropping the G.

Reply:
@oliverchang you mentioned that there may be a preference naming it "@", can you confirm?

Reply:
@oliverchang I changed the tag to "_@_" -- can you confirm the tag you prefer here?

Reply:
I think `@` would be the nicest, but `_@_` is fine too.

Reply:
+1. Can we use `@` as is?

Reply:
yep, fixed.